.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
 python:
     - "2.7"
-    - "3.4"
```

Comment:
I wonder if it's better to comment this out with a note that romanesco is what is preventing us from using 3.4, or maybe better in the docs?